<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Output</class>
 <widget class="QDialog" name="Output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFmpeg Output</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>480</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
    <widget class="QWidget" name="advOutRecFFmpeg">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignLeft|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>FFmpeg.Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeft|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>advOutFFType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="advOutFFType">
        <item>
         <property name="text">
          <string>FFmpeg.Type.RecordToFile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFmpeg.Type.URL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>FFmpeg.SavePathURL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_1">
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="advOutFFRecPath">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="advOutFFPathBrowse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="advOutFFURL">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="advOutFFDeviceList"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="advOutFFNoSpace">
        <property name="text">
         <string>FFmpeg.NoSpaceFileName</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>FFmpeg.Filename</string>
        </property>
        <property name="buddy">
         <cstring>filenameFormatting</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="filenameFormatting"/>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="overwriteIfExists">
        <property name="text">
         <string>FFmpeg.OverwriteIfExists</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>FFmpeg.Format</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFFormat</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="advOutFFFormat"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>FFmpeg.FormatDesc</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="advOutFFFormatDesc">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>FFmpeg.MuxerSettings</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFMCfg</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="advOutFFMCfg"/>
      </item>    
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>FFmpeg.VideoBitrate</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFVBitrate</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="advOutFFVBitrate">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>2500</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>FFmpeg.GOPSize</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFVGOPSize</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="advOutFFVGOPSize">
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="advOutFFUseRescale">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>FFmpeg.Rescale</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="advOutFFRescale">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QCheckBox" name="advOutFFIgnoreCompat">
        <property name="text">
         <string>FFmpeg.IgnoreCodecCompat</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_12">
       <property name="text">
        <string>FFmpeg.VEncoder</string>
       </property>
       <property name="buddy">
        <cstring>advOutFFVEncoder</cstring>
       </property>
      </widget>
     </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="advOutFFVEncoder"/>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>FFmpeg.VEncoderSettings</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFVCfg</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="advOutFFVCfg"/>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>FFmpeg.AudioBitrate</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFABitrate</cstring>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QSpinBox" name="advOutFFABitrate">
        <property name="minimum">
         <number>32</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>FFmpeg.AudioTrack</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QWidget" name="widget_10" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="advOutFFTrack1">
           <property name="text">
            <string notr="true">1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="advOutFFTrack2">
           <property name="text">
            <string notr="true">2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="advOutFFTrack3">
           <property name="text">
            <string notr="true">3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="advOutFFTrack4">
           <property name="text">
            <string notr="true">4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="advOutFFTrack5">
           <property name="text">
            <string notr="true">5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="advOutFFTrack6">
           <property name="text">
            <string notr="true">6</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>FFmpeg.AEncoder</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFAEncoder</cstring>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QComboBox" name="advOutFFAEncoder"/>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>FFmpeg.AEncoderSettings</string>
        </property>
        <property name="buddy">
         <cstring>advOutFFACfg</cstring>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLineEdit" name="advOutFFACfg"/>
      </item>
     </layout>
    </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>508</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>508</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>508</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startOutput">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>508</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopOutput">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>508</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>advOutFFRecPath</tabstop>
  <tabstop>advOutFFPathBrowse</tabstop>
  <tabstop>advOutFFURL</tabstop>	 
  <tabstop>filenameFormatting</tabstop>
  <tabstop>overwriteIfExists</tabstop>
  <tabstop>advOutFFFormat</tabstop>
  <tabstop>advOutFFVBitrate</tabstop>
  <tabstop>advOutFFVGOPSize</tabstop>
  <tabstop>advOutFFUseRescale</tabstop>
  <tabstop>advOutFFRescale</tabstop>
  <tabstop>advOutFFIgnoreCompat</tabstop>
  <tabstop>advOutFFVEncoder</tabstop>
  <tabstop>advOutFFVCfg</tabstop>
  <tabstop>advOutFFABitrate</tabstop>
  <tabstop>advOutFFTrack1</tabstop>
  <tabstop>advOutFFTrack2</tabstop>
  <tabstop>advOutFFTrack3</tabstop>
  <tabstop>advOutFFTrack4</tabstop>
  <tabstop>advOutFFTrack5</tabstop>
  <tabstop>advOutFFTrack6</tabstop>
  <tabstop>advOutFFAEncoder</tabstop>
  <tabstop>advOutFFACfg</tabstop>
  <tabstop>advOutFFType</tabstop>
  <tabstop>advOutFFMCfg</tabstop>
  <tabstop>advOutFFNoSpace</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>advOutFFType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advOutFFUseRescale</sender>
   <signal>toggled(bool)</signal>
   <receiver>advOutFFRescale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
